app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -536,21 +536,19 @@ private void forceSuggestions() {
     }
   }
 
-  private Observable<Void> recyclerViewReachedBottom(RecyclerView recyclerView) {
+  private Observable<Object> recyclerViewReachedBottom(RecyclerView recyclerView) {
```

Comment:
This Observable<Object> looks really weird and doesn't seem to have a purpose. Maybe keep the Void observable? I.e. map it to null.